sematic/tests/fixtures.py

```diff
@@ -31,6 +32,16 @@ def valid_client_version():
         api_client._validated_client_version = current_validated_client_version
 
 
+@pytest.fixture
+def allow_any_run_state_transition():
```

Comment:
Is this done because existing tests use illegal transitions?
Please add todos to all these tests to fix them.

Reply:
It's not really problematic that they do. Basically, for purposes of a test you might want to magically create a run that's already in the RESOLVED state or something, without *actually* moving the run through all the intervening states. This enables that use case.